src/chapters/interfaces-and-polymorphism/index.rst

```diff
@@ -0,0 +1,11 @@
+Interfaces and Polymorphism
+===========================
+
+.. toctree::
+   :maxdepth: 3
+
+   introduction
+   interfaces
+   interfaces-and-abstract-classes
+   given-interfaces
+   testing-interfaces
```

Comment:
VSCode is flagging "no newline at end of file".

Reply:
Done! Thanks!